benchmarks/timeseries/benchmark_datasourcing.py

```diff
@@ -70,7 +70,7 @@ async def benchmark_data_sourcing(
         num_ev_chargers * len(COMPONENT_METRIC_IDS) * num_msgs_per_battery
     )
     mock_grid = MockMicrogrid(
-        grid_side_meter=False, num_values=num_msgs_per_battery, sample_rate_s=0.0
+        main_meter=False, num_values=num_msgs_per_battery, sample_rate_s=0.0
```

Comment:
if you're adding a `add_consumer_meters`, then you might as well add a `add_grid_meters` method and drop the flag.  There has already been some talk about locations with multiple grid meters anyway.

Reply:
that' s the next step actually

Reply:
I feel like this is not a good idea,  because this makes our tests rely on the implementation rather than on just the expected behaviour.  I think instead we should always add a main meter,  and have special cases for graphs without main meters.  Maybe we can discuss tomorrow during the meeting.